lib/Centreon/Object/Relation/Host/Template/Host.php

```diff
@@ -70,6 +70,30 @@ public function insert($fkey, $skey = null)
         $this->db->query($sql, array($fkey, $skey, $order));
     }
 
+    /**
+     * Delete host template / host relation and linked services
+     *
+     * @param int $fkey - host template
+     * @param int $skey - host
+     * @return void
+     */
+    public function delete($fkey, $skey = null): void
+    {
+        global $pearDB;
+        $this->db->beginTransaction();
+        try {
+            parent::delete($fkey, $skey);
+            $pearDB = $this->db;
+            $centreon = true; // Needed so we can include file below
+            require_once __DIR__."/../../../../../../www/include/configuration/configObject/host/DB-Func.php";
```

Comment:
maybe will be better if you replace `__DIR__."/../../../../../../` with `_CENTREON_PATH_` constant, how do you think?

Reply:
+1

Reply:
style : please add spaces around the dot